cmd/thanos/config.go

```diff
@@ -69,24 +70,10 @@ func (pc *prometheusConfig) registerFlag(cmd extkingpin.FlagClause) *prometheusC
 	cmd.Flag("prometheus.ready_timeout",
 		"Maximum time to wait for the Prometheus instance to start up").
 		Default("10m").DurationVar(&pc.readyTimeout)
+	pc.httpClient = extkingpin.RegisterHTTPConfigFlags(cmd)
 	return pc
 }
 
-type connConfig struct {
-	maxIdleConns        int
-	maxIdleConnsPerHost int
-}
-
-func (cc *connConfig) registerFlag(cmd extkingpin.FlagClause) *connConfig {
```

Comment:
Let's make sure we add this move to the changelog

Reply:
Why it was receive? 🤔

Reply:
By mistake I assume :thinking: I don't see it being used anywhere